2020-04-17 18:26:16,536 INFO    MainThread:10256 [wandb_setup.py:_log_setup():205] Logging to wandb\wandb-20200417_182616-10256\debug.log
2020-04-17 18:26:16,537 INFO    MainThread:10256 [wandb_setup.py:_setup():231] multiprocessing start_methods=spawn
2020-04-17 18:26:16,553 INFO    MainThread:10256 [wandb_init.py:init():303] runconfig: {'_wandb': {'desc': None, 'value': {'cli_version': '0.0.14'}}}
2020-04-17 18:26:18,301 ERROR   MainThread:10256 [wandb_init.py:init():392] we got issues
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 389, in init
    run = wi.init()
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 306, in init
    ret = backend.interface.send_run_sync(r, timeout=30)
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\interface\interface.py", line 356, in send_run_sync
    resp = self._request_response(req)
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\interface\interface.py", line 329, in _request_response
    rsp = self.response_queue.get(timeout=timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\queues.py", line 104, in get
    if not self._poll(timeout):
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 257, in poll
    return self._poll(timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 330, in _poll
    return bool(wait([self], timeout))
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 868, in wait
    ready_handles = _exhaustive_wait(waithandle_to_obj.keys(), timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 800, in _exhaustive_wait
    res = _winapi.WaitForMultipleObjects(L, False, timeout)
KeyboardInterrupt
2020-04-17 18:26:18,308 WARNING MainThread:10256 [wandb_init.py:init():401] interupted
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 389, in init
    run = wi.init()
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 306, in init
    ret = backend.interface.send_run_sync(r, timeout=30)
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\interface\interface.py", line 356, in send_run_sync
    resp = self._request_response(req)
  File "C:\ProgramData\Anaconda3\lib\site-packages\wandb\interface\interface.py", line 329, in _request_response
    rsp = self.response_queue.get(timeout=timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\queues.py", line 104, in get
    if not self._poll(timeout):
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 257, in poll
    return self._poll(timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 330, in _poll
    return bool(wait([self], timeout))
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 868, in wait
    ready_handles = _exhaustive_wait(waithandle_to_obj.keys(), timeout)
  File "C:\ProgramData\Anaconda3\lib\multiprocessing\connection.py", line 800, in _exhaustive_wait
    res = _winapi.WaitForMultipleObjects(L, False, timeout)
KeyboardInterrupt
